modules/network-annotation-processor/src/main/java/org/apache/ignite/internal/network/processor/serialization/MessageWriterMethodResolver.java

```diff
@@ -78,6 +78,7 @@ CodeBlock resolveWriteMethod(ExecutableElement getter) {
 
         if (getter.getAnnotation(Marshallable.class) != null) {
             getterName = addByteArrayPostfix(getterName);
+            propertyName = addByteArrayPostfix(propertyName);
```

Comment:
Is there a test that reproduces the problem?
If not, is it possible to add one?

Reply:
I added a test